leakcanary-analyzer/src/main/java/com/squareup/leakcanary/AnalysisResult.java

```diff
@@ -56,16 +57,23 @@ public static AnalysisResult failure(Exception exception, long analysisDurationM
   /** Null unless the analysis failed. */
   public final Exception failure;
 
+  /**
+   * The number of bytes which would be freed if all references to the leaking object were
+   * released. 0 if {@link #leakFound} is false.
+   */
+  public final long retainedHeapSize;
```

Comment:
It's probably worth a note that this is an estimate. For example Bitmaps on some versions of Android (it has gone back and forth a few times) have native GC roots. As a result MAT tends to attribute their byte size to the native root, rather than to the objects that actually reference the Bitmap object.

In the long run you could make this even better by detecting when objects reference Bitmaps and counting that here, rather than just relying on what the analyzer gives you.